Updating Store.json for Liverig 5.0.0 changes

VictorARMota opened the pull request:
Added information about fetching Opcua objects as fields for Live.

Commit:
Updating Store.json for Liverig 5.0.0 changes

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+
+![img.png](../../.gitbook/assets/OpcuaObjectEvent4.png)
+
+**OBS**: Since the `tags` field from the `store.json` file is a `Map`, you need to add a new alias for each field you want to fetch. Example:
```
```suggestion
**Note**: Since the `tags` field from the `store.json` file is a `Map`, you need to add a new alias for each field you want to fetch. Example:
```

amadeubarbosa committed:
Replace an "OBS" by a "Note"

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+
+
+{% hint style="info" %} 
+Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to easily change settings.
```
```suggestion
Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to change the settings easily.
```

amadeubarbosa committed:
Minor improvement on the info panel

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
 Obs: For **OPC-UA** sources, the tag field should be written as the following pattern: `ns=<namespaceindex>;<type>=<value>`
 
-Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to do this settings.
+### For Liverig Collector versions 5.0.0 or above
```
```suggestion
## LiveRig Collector 5.0.0 or above
```

amadeubarbosa committed:
Update collector/configuration/store.json.md

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
-Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to do this settings.
+## LiveRig Collector 5.0.0 or above
+
+Since Liverig Collector version 5.0.0, the collector can be configured to extract field from object values in **OPC-UA** sources. 
```
```suggestion
Since LiveRig Collector version 5.0.0, the collector can be configured to extract field from object values in **OPC-UA** sources. 
```

amadeubarbosa committed:
LiveRig spelling

amadeubarbosa committed:
Explain the OPC nature of this file

- includes an introduction to inform is used only for OPC
- includes an brief about opc to witsml conversion

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -28,9 +30,11 @@ The Liverig collector also requires the Node Ids (Tags) values, among other info
-Each object under rigs is related to an **OPC-DA** or **OPC-UA** Source, linking the `store.json` and `sources.xml` files through their Rig Name. Some generic fields configure global collector settings: database, endpoint, and limit. These configurations are required if the **OPC** Source is going through the **OPC** to **WITSML** conversion.
+Each object under rigs is related to an **OPC-DA** or **OPC-UA** source, linking the `store.json` and `sources.xml` files through their **Rig Name**. 
+
+Some extra fields are responsible for an additional collector feature known as [**OPC to WITSML converter**](./../protocol-conversion.md). These optional fields are: `database`, `endpoint`, and `limit`. Once the `endpoint` and `database` are configured, a basic WITSML server will start backed by a PostgreSQL database to keep the data and enable the WITSML queries on top of it.
```
```suggestion
Some extra fields are responsible for an additional collector feature known as [OPC to WITSML protocol conversion](./../protocol-conversion.md). These optional fields are: `database`, `endpoint`, and `limit`. Once the `endpoint` and `database` are configured, a basic WITSML server will start backed by a PostgreSQL database to keep the data and enable the WITSML queries on top of it.
```

amadeubarbosa committed:
Removes the bold over the opc to witsml conversion

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -40,6 +44,71 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+`{tag}?field={path}`.
+
+So, in this example the previous tag `"ns=2;s=HelloWorld/ScalarTypes/UtcTime"` would be changed to `"ns=2;s=HelloWorld/ScalarTypes/UtcTime?field=/utcTime"`
+Resulting in the following value:
+
+![OpcuaObjectEvent2.png](../../.gitbook/assets/OpcuaObjectEvent2.png)
```
```suggestion

Resulting in the following value:

![Extracting the timestamp from an OPC date time object](../../.gitbook/assets/OpcuaObjectEvent2.png)
```

amadeubarbosa committed:
Caption for the image

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -40,6 +44,72 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+
+**Example 1:**
+
+![OpcuaObjectEvent1.png](../../.gitbook/assets/OpcuaObjectEvent1.png)
```
```suggestion
![A date time OPC object arrives as is](../../.gitbook/assets/OpcuaObjectEvent1.png)
```

amadeubarbosa committed:
Caption for the image

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -40,6 +44,72 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+
+Resulting in the following value:
+
+![img.png](../../.gitbook/assets/OpcuaObjectEvent4.png)
```
```suggestion
![Extracting the encoded object](../../.gitbook/assets/OpcuaObjectEvent4.png)
```

amadeubarbosa commented on collector/configuration/store.json.md:
```diff
@@ -40,6 +44,72 @@ The alias is used as a key reference for tags, units and types values.&#x20;
+
+**Example 2:**
+
+![OpcuaObjectEvent3.png](../../.gitbook/assets/OpcuaObjectEvent3.png)
```
```suggestion
![An encoded OPC object arrives as is](../../.gitbook/assets/OpcuaObjectEvent3.png)
```

amadeubarbosa committed:
Caption for images
